src/datahike/db.cljc

```diff
@@ -521,6 +525,7 @@
                  create-avet (fn [a v e tx] (Datom. e a v tx true))
                  ]
              (case-tree [e a (some? v) tx]
+                        ;;rb: shorter datom syntax
```

Comment:
We only need to pass vectors for datoms there. The Datom construction is unnecessary, I was exploring to store datoms in the index instead of flat vectors, but I think the latter is better (because serialization is more compact and easier that way).